[2023-05-07T03:19:57.448+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-07T03:19:57.458+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-07T03:19:57.459+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-07T03:19:57.460+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-07T03:19:57.461+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-07T03:19:57.479+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): scrape_teams_9-16> on 2022-05-02 01:00:00+00:00
[2023-05-07T03:19:57.485+0000] {standard_task_runner.py:55} INFO - Started process 1354 to run task
[2023-05-07T03:19:57.490+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_to_csv', 'scrape_teams_9-16', 'scheduled__2022-05-02T01:00:00+00:00', '--job-id', '2758', '--raw', '--subdir', 'DAGS_FOLDER/scrape_and_push_dag.py', '--cfg-path', '/tmp/tmp6ht948jb']
[2023-05-07T03:19:57.493+0000] {standard_task_runner.py:83} INFO - Job 2758: Subtask scrape_teams_9-16
[2023-05-07T03:19:57.585+0000] {task_command.py:389} INFO - Running <TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [running]> on host 6589836e9c08
[2023-05-07T03:19:57.668+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=scrape_and_save_to_csv
AIRFLOW_CTX_TASK_ID=scrape_teams_9-16
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T01:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-02T01:00:00+00:00
[2023-05-07T03:19:57.691+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: scrape_and_save() missing 1 required positional argument: 'current_date'
[2023-05-07T03:19:57.701+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=scrape_and_save_to_csv, task_id=scrape_teams_9-16, execution_date=20220502T010000, start_date=20230507T031957, end_date=20230507T031957
[2023-05-07T03:19:57.710+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 2758 for task scrape_teams_9-16 (scrape_and_save() missing 1 required positional argument: 'current_date'; 1354)
[2023-05-07T03:19:57.742+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-07T03:19:57.796+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-08T23:43:14.401+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-08T23:43:14.410+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-08T23:43:14.411+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-08T23:43:14.414+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-08T23:43:14.415+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-08T23:43:14.431+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): scrape_teams_9-16> on 2022-05-02 01:00:00+00:00
[2023-05-08T23:43:14.437+0000] {standard_task_runner.py:55} INFO - Started process 153 to run task
[2023-05-08T23:43:14.441+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_to_csv', 'scrape_teams_9-16', 'scheduled__2022-05-02T01:00:00+00:00', '--job-id', '3100', '--raw', '--subdir', 'DAGS_FOLDER/scrape_and_push_dag.py', '--cfg-path', '/tmp/tmpnchvh6iy']
[2023-05-08T23:43:14.443+0000] {standard_task_runner.py:83} INFO - Job 3100: Subtask scrape_teams_9-16
[2023-05-08T23:43:14.523+0000] {task_command.py:389} INFO - Running <TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [running]> on host f41dbfa3d325
[2023-05-08T23:43:14.560+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-08T23:43:14.609+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=scrape_and_save_to_csv
AIRFLOW_CTX_TASK_ID=scrape_teams_9-16
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T01:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-02T01:00:00+00:00
[2023-05-08T23:43:14.613+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/helpers.py", line 59, in scrape_and_save
    after = datetime.fromisoformat(f"{{ execution_date }}")
ValueError: Invalid isoformat string: '{ execution_date }'
[2023-05-08T23:43:14.626+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=scrape_and_save_to_csv, task_id=scrape_teams_9-16, execution_date=20220502T010000, start_date=20230508T234314, end_date=20230508T234314
[2023-05-08T23:43:14.635+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 3100 for task scrape_teams_9-16 (Invalid isoformat string: '{ execution_date }'; 153)
[2023-05-08T23:43:14.654+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-08T23:43:14.683+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-08T23:46:57.378+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-08T23:46:57.390+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-08T23:46:57.392+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-08T23:46:57.394+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-08T23:46:57.396+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-08T23:46:57.409+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): scrape_teams_9-16> on 2022-05-02 01:00:00+00:00
[2023-05-08T23:46:57.416+0000] {standard_task_runner.py:55} INFO - Started process 560 to run task
[2023-05-08T23:46:57.420+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_to_csv', 'scrape_teams_9-16', 'scheduled__2022-05-02T01:00:00+00:00', '--job-id', '3206', '--raw', '--subdir', 'DAGS_FOLDER/scrape_and_push_dag.py', '--cfg-path', '/tmp/tmpzio_od5b']
[2023-05-08T23:46:57.424+0000] {standard_task_runner.py:83} INFO - Job 3206: Subtask scrape_teams_9-16
[2023-05-08T23:46:57.513+0000] {task_command.py:389} INFO - Running <TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [running]> on host f41dbfa3d325
[2023-05-08T23:46:57.543+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-08T23:46:57.592+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=scrape_and_save_to_csv
AIRFLOW_CTX_TASK_ID=scrape_teams_9-16
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T01:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-02T01:00:00+00:00
[2023-05-08T23:46:57.596+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/helpers.py", line 59, in scrape_and_save
    after = datetime.fromisoformat(f"{{ execution_date }}")
ValueError: Invalid isoformat string: '{ execution_date }'
[2023-05-08T23:46:57.621+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=scrape_and_save_to_csv, task_id=scrape_teams_9-16, execution_date=20220502T010000, start_date=20230508T234657, end_date=20230508T234657
[2023-05-08T23:46:57.632+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 3206 for task scrape_teams_9-16 (Invalid isoformat string: '{ execution_date }'; 560)
[2023-05-08T23:46:57.675+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-08T23:46:57.701+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-09T21:54:03.659+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-09T21:54:03.682+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [queued]>
[2023-05-09T21:54:03.683+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-09T21:54:03.686+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-09T21:54:03.701+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-09T21:54:03.732+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): scrape_teams_9-16> on 2022-05-02 01:00:00+00:00
[2023-05-09T21:54:03.741+0000] {standard_task_runner.py:55} INFO - Started process 13005 to run task
[2023-05-09T21:54:03.753+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_to_csv', 'scrape_teams_9-16', 'scheduled__2022-05-02T01:00:00+00:00', '--job-id', '3438', '--raw', '--subdir', 'DAGS_FOLDER/scrape_and_push_dag.py', '--cfg-path', '/tmp/tmpsov36n8j']
[2023-05-09T21:54:03.758+0000] {standard_task_runner.py:83} INFO - Job 3438: Subtask scrape_teams_9-16
[2023-05-09T21:54:03.893+0000] {task_command.py:389} INFO - Running <TaskInstance: scrape_and_save_to_csv.scrape_teams_9-16 scheduled__2022-05-02T01:00:00+00:00 [running]> on host f41dbfa3d325
[2023-05-09T21:54:03.956+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-09T21:54:04.008+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=scrape_and_save_to_csv
AIRFLOW_CTX_TASK_ID=scrape_teams_9-16
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T01:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-02T01:00:00+00:00
[2023-05-09T21:54:10.250+0000] {helpers.py:94} INFO - Detroit Pistons: Successfully scraped!
[2023-05-09T21:54:24.581+0000] {helpers.py:94} INFO - Golden State Warriors: Successfully scraped!
[2023-05-09T21:54:40.121+0000] {helpers.py:94} INFO - Houston Rockets: Successfully scraped!
[2023-05-09T21:54:54.737+0000] {helpers.py:94} INFO - Indiana Pacers: Successfully scraped!
[2023-05-09T21:55:10.844+0000] {helpers.py:94} INFO - Los Angeles Clippers: Successfully scraped!
[2023-05-09T21:55:26.601+0000] {helpers.py:94} INFO - Los Angeles Lakers: Successfully scraped!
[2023-05-09T21:55:42.277+0000] {helpers.py:94} INFO - Memphis Grizzlies: Successfully scraped!
[2023-05-09T21:55:55.354+0000] {helpers.py:94} INFO - Miami Heat: Successfully scraped!
[2023-05-09T21:55:55.382+0000] {helpers.py:109} INFO - Saved in /opt/***/data/output_scrape_teams_9-16.csv
[2023-05-09T21:55:55.385+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-09T21:55:55.409+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=scrape_and_save_to_csv, task_id=scrape_teams_9-16, execution_date=20220502T010000, start_date=20230509T215403, end_date=20230509T215555
[2023-05-09T21:55:55.530+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-09T21:55:55.566+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
